_translator-files/po/de/FAQ.po

```diff
@@ -45,159 +47,161 @@ msgid ""
-msgstr ""
+msgstr "Wenn jemand einen Server in eines der öffentlichen Verzeichnisse einträgt, die Jamulus standardmäßig bereitstellt, akzeptiert er damit, dass jeder auf ihm spielen kann. Jamulus hat keinen Passwortschutz oder andere Authentifizierungsmechanismen eingebaut. Einige Server können jedoch in der Willkommensnachricht, die du im Chat-Fenster sehen kannst, Richtlinien angeben."
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:34
 msgid "Note also that Servers do not have to be registered on a Directory in order for Jamulus Clients to connect to them. Server operators can simply give out the address of their Servers to those they want to play with, and they can then connect directly. Have a look at [this overview for more information](/wiki/Running-a-Server#server-types)."
-msgstr ""
+msgstr "Beachte auch, dass die Server nicht in einem Verzeichnis registriert sein müssen, damit Jamulus-Clients sich mit ihnen verbinden können. Serverbetreiber können einfach die Adresse deines Servers an diejenigen weitergeben, mit denen sie spielen wollen, und diese können sich dann direkt verbinden. Schauen dir [diese Übersicht für weitere Informationen](/wiki/Running-a-Server#Server Typen) an."
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:36
 msgid "### Why shouldn’t I use wireless equipment?"
-msgstr ""
+msgstr "### Warum sollte ich keine drahtlosen Geräte verwenden?"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:38
 msgid "Whether or not you will be able to play in time with other musicians depends mainly on how much latency (delay) you have in your sound signal and whether you are all following [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together). An overall delay much bigger than 50 ms usually makes it too hard to play in time unless you train yourself to do so. Some musicians report they can play in time with delays of up to 70 ms by playing ahead of the beat. But much beyond that and it becomes too difficult unless timing isn’t an important factor in the music or you just want to listen."
-msgstr ""
+msgstr "Ob du im Takt mit anderen Musikern spielen kannst, hängt hauptsächlich davon ab, wie viel Latenz (Verzögerung) du in deinem Tonsignal hast und ob alle die [Goldene Regel](/wiki/Client-Troubleshooting#der-sound-ist-gut-aber-es-ist-schwierig-im-Takt-zu-bleiben) befolgen. Bei einer Gesamtverzögerung von mehr als 50 ms ist es in der Regel zu schwierig, im Takt zu spielen, es sei denn, du trainierst dich selbst dafür. Einige Musiker berichten, dass sie mit einer Verzögerung von bis zu 70 ms im Takt spielen können, indem sie vor dem Beat spielen. Aber viel darüber hinaus wird es zu schwierig, es sei denn, das Timing ist kein wichtiger Faktor in der Musik oder du willst nur zuhören."
```

Comment:
@gilgongo I think we should make this text a bit more concise. WiFi = more latency.